src/VisualStudio/Core/Def/Implementation/Snippets/AbstractSnippetExpansionClient.cs

```diff
@@ -393,7 +451,338 @@ public virtual bool TryInsertExpansion(int startPositionInSubjectBuffer, int end
                 iEndIndex = endIndex
             };
 
-            return expansion.InsertExpansion(textSpan, textSpan, this, LanguageServiceGuid, out ExpansionSession) == VSConstants.S_OK;
+            if (expansion.InsertExpansion(textSpan, textSpan, this, LanguageServiceGuid, out ExpansionSession) == VSConstants.S_OK)
+            {
+                // This expansion is not derived from a symbol
+                _symbols = default;
+                _symbol = null;
+                _arguments = _arguments.Clear();
```

Comment:
5th time.

Reply:
One for each case where `ExpansionSession` is set

➡️ Tracking state containment as https://github.com/dotnet/roslyn/pull/49916#discussion_r544816693